packages/svelte/src/containers/xy-container/xy-container.svelte

```diff
@@ -5,6 +5,8 @@
   type Datum = $$Generic
 
   export let data: Datum[]
+  export let className = ''
```

Comment:
@reb-dev Do you know if there's a way to add comments to svelte props an make them show up as you type your html? That would be great to add your note on using `:global` here to make it more obvious to the users.

Reply:
@rokotyan Yes! I've just added JS docs in the last push. Couldn't get the same level as detail as above but I think it gets the point across anyway.

Reply:
@reb-dev Awesome, thanks!

Reply:
@rokotyan Nevermind, I guess it broke the build. Looking into alternatives